### ISSUE TITLE
[BUG] Some examples use deprecated Reax

### ISSUE THREAD

**Issue by stanmoore1** [lammps/lammps#1337]
**Summary**

The following examples use the deprecated `pair reax`, which has been removed from LAMMPS:

```
examples/USER/qtb/methane_qtb', 'methane_qtb'
examples/USER/qtb/methane_qbmsst', 'methane_qbmsst'
```

**LAMMPS Version and Platform**

Latest Master

Should these examples be removed, or be updated to use the `USER-REAXC` package?

**Comment by akohlmey** [comment]
Updated



--
Dr. Axel Kohlmeyer  akohlmey@gmail.com  http://goo.gl/1wk0
College of Science & Technology, Temple University, Philadelphia PA, USA
International Centre for Theoretical Physics, Trieste. Italy.

**Comment by evoyiatzis** [comment]
For the case of "examples/USER/qtb/methane_qtb", if line 47 in methane_qtb.in is replaced with:

pair_style              reax/c NULL #10.0 1 1 1.0e-6
pair_coeff              * * ffield.reax C H
fix                     11 all qeq/reax 1 0.0 10.0 1.0e-6 reax/c

then the example works fine. The same replacement has to be made in file "methane_qtb.mod" line 41 for the second example.
